bg-find-sync/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG  DOCKER_REPOSITORY
+ARG  MANAGE_COURSES_BACKEND_IMAGE
+ARG  CODE_VERSION=latest
+
+FROM ${DOCKER_REPOSITORY}/${MANAGE_COURSES_BACKEND_IMAGE}:${CODE_VERSION}
+
+CMD bundle exec rails db:migrate && bundle exec bin/delayed_job --queue=find_sync run
```

Comment:
should this do migrate? and not the actual site....

Reply:
> should this do migrate? and not the actual site....

I think it's ok to do the migrate here, but we'll have to see if there's any race conditions. I'd rather not require the web app to be started before this container can be started.